site/website/blog/2020-07-16-designing-evergreen-cache-cdc.md

```diff
@@ -270,28 +268,26 @@ The pipeline definition is quite straightforward:
 ```java
 pipeline.readFrom(source)                                       //1
         .withoutTimestamps()
-        .map(r -> {
-            Person person = r.value().toObject(Person.class);   //2
-            return Util.entry(person.id, person);               //3
-        })
-        .writeTo(Sinks.remoteMap(                               //4
-                "entities",                                     //5
-                new CustomClientConfig(env.get("CACHE_HOST"))   //6
+        .writeTo(CdcSinks.remoteMap(                            //2
+                "entities",                                     //3
+                new CustomClientConfig(env.get("CACHE_HOST")),  //4
+                r -> r.key().toMap().get("id"),                 //5
+                r -> r.value().toObject(Person.class)           //6
         ));
 ```
 
-1. Get a stream of Jet `ChangeRecord`
+1. Get a stream of Jet `ChangeRecord` items
 
-2. Convert `ChangeRecord` to a regular `Person` POJO
+2. Create the sink to write to, a remote map
```

Comment:
No comma

Reply:
I don't think it's ok to remove the comma: https://owl.purdue.edu/owl/general_writing/punctuation/commas/commas_with_nonessential_elements.html

Reply:
"Create a sink to write to a remote map" parses differently than "Create a sink to write to, a remote map". But you could say "Create a sink to write to (a remote map)".

Reply:
Ahh, now I understand what Viliam was saying. Yes, I did actually intend to say `Create a sink to write to, a remote map`, but now I realise that `Create a sink to write to a remote map` makes much more sense, even though it is not what I initially intended. Will fix.